baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/DangerousStringInternUsage.java

```diff
@@ -33,30 +33,33 @@
-                + "implementations such as Guava Interners "
-                + "(https://guava.dev/releases/27.0.1-jre/api/docs/com/google/common/collect/Interners.html). If you "
-                + "are confident that String.intern is the correct tool, please make sure you fully understand the "
-                + "consequences."
-                + "\nFrom https://shipilev.net/jvm/anatomy-quarks/10-string-intern/"
-                + "\n> For OpenJDK, String.intern() is the gateway to native JVM String table, and it comes with"
-                + "\n> caveats: throughput, memory footprint, pause time problems will await the users. It is very"
-                + "\n> easy to underestimate the impact of these caveats. Hand-rolled deduplicators/interners are"
-                + "\n> working much more reliably, because they are working on Java side, are just the regular Java"
-                + "\n> objects, generally better sized/resized, and also can be thrown away completely when not needed"
-                + "\n> anymore. GC-assisted String deduplication does alleviate things even more."
-                + "\n> In almost every project we were taking care of, removing String.intern() from the hotpaths, "
-                + "\n> or optionally replacing it with a handrolled deduplicator, was the very profitable performance"
-                + "\n> optimization. Do not use String.intern() without thinking very hard about it, okay?")
+        summary =
+                "Should not use String.intern(). Java string intern is complex and unpredictable. In most cases intern"
+                    + " performs worse than pure-java implementations such as Guava Interners"
+                    + " (https://guava.dev/releases/27.0.1-jre/api/docs/com/google/common/collect/Interners.html). If"
+                    + " you are confident that String.intern is the correct tool, please make sure you fully"
+                    + " understand the consequences.\n"
+                    + "From https://shipilev.net/jvm/anatomy-quarks/10-string-intern/\n"
+                    + "> For OpenJDK, String.intern() is the gateway to native JVM String table, and it comes with\n"
+                    + "> caveats: throughput, memory footprint, pause time problems will await the users. It is very\n"
+                    + "> easy to underestimate the impact of these caveats. Hand-rolled deduplicators/interners are\n"
+                    + "> working much more reliably, because they are working on Java side, are just the regular"
+                    + " Java\n"
+                    + "> objects, generally better sized/resized, and also can be thrown away completely when not"
+                    + " needed\n"
+                    + "> anymore. GC-assisted String deduplication does alleviate things even more.\n"
+                    + "> In almost every project we were taking care of, removing String.intern() from the hotpaths,"
+                    + " \n"
```

Comment:
oof, this is nasty

Reply:
yeah... I agree, should track this via an issue.
I thought that https://github.com/palantir/palantir-java-format/pull/35 would have fixed this sort of issue, but doesn't look like it sadly.